### PR TITLE
PHRAS-3282 refacto worker events/messages/queues

### PR DESCRIPTION
**Pull request by jygaulier**
WIP DO NOT MERGE
TODO
- rewrite "loop" q to remove "if"
- fix front admin/ux
- check "loop" q (eg. validationReminder) : loop delay ? admin ux
- ? move AMQPConnection::MESSAGES to objects ?
DONE
- removed most AMQPConnection consts

### CODE REVIEW

File: lib/Alchemy/Phrasea/WorkerManager/Form/QueueSettingsType.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 use Alchemy\Phrasea\WorkerManager\Queue\AMQPConnection;
 use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+uuse Symfony\Component\Form\Extension\Core\Type\IntegerType;
```
Comment (nmaillat) [inline lib/Alchemy/Phrasea/WorkerManager/Form/QueueSettingsType.php:8]: uuse ? typo ?

File: lib/conf.d/configuration.yml [hunk 2]
```diff
@@ -321,6 +321,38 @@ workers:
             user: ''
             password: ''
             vhost: /
+    queues:
```
Comment (nmaillat) [inline lib/conf.d/configuration.yml:324]: update lib/conf.d/configuration.yml ?

File: lib/Alchemy/Phrasea/WorkerManager/Form/QueueSettingsType.php (hunk 1 above)
Comment (jygaulier) [reply lib/Alchemy/Phrasea/WorkerManager/Form/QueueSettingsType.php:8]: typo !

**Comment by nmaillat** [comment]
cose in favor of #3680

**Pull request closed without merging**